projects/question-editor-library/src/lib/components/answer/answer.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="sb-field-group mt-15">
-  <label class="fs-0-92">Answers<span class="sb-required">*</span></label><i class="icon info circle ml-2" suiPopup popupDelay="500" popupText="User can add their answers here"></i>
+  <label class="fs-0-92">Answers<span class="sb-required">*</span></label><i class="icon info circle ml-2" suiPopup popupDelay="500" popupText="Please provide an answer for the question Check preview to understand how it would look."></i>
```

Comment:
Please provide an answer for the question. Check preview to understand how it would look.

Reply:
Done